### ISSUE TITLE
Fine tuning AIMNet2

### ISSUE THREAD

**Issue by MorrowChem** [isayevlab/AIMNet2#23]
Hi, I'm interested in fine-tuning AIMNet2 on my particular system. 

I want to make sure additional DFT reference calculations are consistent with those used to train the original model, but I can't find any training data labelled with your reference method to which I can compare.

Is there any chance you could include these labels in a small test set in this repo, if they're not publicly available already?

**Comment by zubatyuk** [comment]
Check this out: [zubatyuk/aimnet2](https://github.com/zubatyuk/aimnet2)
The documentation is sparse at the moment: [readme.md](https://github.com/zubatyuk/aimnet2/blob/master/aimnet/train/train.md). But the code should work. Please try this out and ask if you have any problems.

**Comment by MorrowChem** [comment]
Thanks, all works well for me
